### ISSUE TITLE
[RTL] The search suggestions and search engines on the URL bar are LTR

### ISSUE THREAD

**Issue by SimonBasca** [mozilla-mobile/firefox-ios#6543]
See screenshot:
https://drive.google.com/file/d/1oBJ9ueHO2Byi3jYk3HTgoKXQ5f1uemAX/view

The search suggestions should start from right to left, and the associated search engine should be on the right.
On the bottom, the search engines should start from right to left, and the magnifying glass should be on the right.

Via : https://bugzilla.mozilla.org/show_bug.cgi?id=1634933

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-2398)

**Comment by data-sync-user** [comment]
➤ Simion Basca commented:

Verifying as fix on main 6aaca8a5c4ba01029db4239fffe6fee63465c111
